components/trades/NewTrade.tsx

```diff
@@ -252,10 +252,14 @@ export default function NewTrade({
   const toggleTeam = (team: string) => {
     setTeams((currentValue) => {
       const teamIndex: number = currentValue.indexOf(team)
-      teamIndex === -1
-        ? currentValue.push(team)
-        : currentValue.splice(teamIndex)
-      return [...currentValue]
+      if(teamIndex === -1) {
+        return [... currentValue, team]
```

Comment:
Is this a syntax error? I think that space between the ellipses and `currentValue` will cause issues.

Reply:
updated with the prettier formatting